src/store/actions.js

```diff
@@ -0,0 +1,15 @@
+import { auth } from "../firebase";
```

Comment:
We're int he Vuex store now, and this file contains all the actions that can be accessed by other components across the app

Seems like we define actions here using const

We're also importing firebase Auth, because the actions we are going to define will be using stuff from Firebase auth